vme/src/experience.cpp

```diff
@@ -95,19 +115,29 @@ int shield_bonus(unit_data *att, unit_data *def, unit_data **pDef_shield)
         }
         else
         {
-            shield_bonus = OBJ_VALUE(def_shield, 1) + OBJ_VALUE(def_shield, 2);
+            shield_bonus = OBJ_VALUE(def_shield, 1) + OBJ_VALUE(def_shield, 2);*/
+/*This code gets shield bonus values and is equivalent to the numbers above but takes into account the shield type itself*/
+        shield_bonus = OBJ_VALUE(def_shield, 0) + OBJ_VALUE(def_shield, 1) + OBJ_VALUE(def_shield, 2); 
```

Comment:
It's a bit odd to add OBJ_VALUE(def_shield, 0) here - as far as I can tell it's the shield type 0,1,2 or 3. I think it's a mistake? +3 won't make much of a difference.